File: JAVA/Daily Flash Codes/Week1/Day4_Jan16/20Aug/prog4.java


import java.io.*;
class Program {

	public static void main(String args[]) throws IOException{
	
	//create object of bufferedRead
	BufferedReader br= new BufferedReader (new InputStreamReader (System.in));
	int row;
	
	System.out.println("Enter Row  num");
	row = Integer.parseInt(br.readLine());
	
	int count = 1;
	for(int olc = 0; olc<row ;olc++) {
		for(int ilc = 0; ilc<row ;ilc++) {
			if(count < 10) 
				System.out.print((count++)+"  " );
			else
				System.out.print((count++)+" " );
		}
		System.out.println();
	}


	}
}

File: C++/Daily Flash/Week2/Day6_Jan26/code/prog4.cpp
#include <iostream>

int main() {
	int row;
	std::cout << "Enter row num: ";
	std::cin >> row;
	int var = 65;
	
	for(int i=0; i<row; i++){
		
		for(int j=0; j<=i; j++) { 
			if(i%2!=0)
				std::cout << (char)var << " ";
			else
			        std::cout << (char)(var+32) << " ";
		}
			std::cout << std::endl;
	}
        return 0;

}
